microsoft-365/admin/add-users/let-users-reset-passwords.md

```diff
@@ -49,7 +49,7 @@ If you found this video helpful, check out the [complete training series for sma
 These steps turn on self-service password reset for everyone in your business.
   
 ::: moniker range="o365-worldwide"
-1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2072756" target="_blank">Settings</a> page.
+1. In the Microsoft 365 admin center at <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">https://admin.microsoft.com</a>, in the left navigation pane, go to the **Settings** \> **Org settings** page.
```

Comment:
Please replace with:
1. In the <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">admin center</a>, go to the **Settings** > **Org settings** page.

Reply:
Thanks @cmcatee-MSFT